src/utils/object.js

```diff
@@ -44,3 +44,26 @@ export function map(opt_initial) {
 export function hasOwn(obj, key) {
   return hasOwn_.call(obj, key);
 }
+
+/**
+ * Returns the value of the property referenced by dot-separated keys.
+ * e.g.
+ * `getPath('a.b[0].c', {a: {b: [{c: 2}]}}) === 2`
+ *
+ * @param {string} path a dot-separated list of keys to reference a value
+ * @param {T} obj a map-like value
+ * @return {*}
+ * @template T
+ */
+export function getPath(path, obj) {
```

Comment:
Sure. I had path first for to make currying with `bind` easy to use to map on an array of objects, but I'm not doing that anywhere.